sensor_msgs/test/test_point_cloud_msg_wrapper.cpp

```diff
@@ -115,7 +117,7 @@ using AllPointTypes = ::testing::Types<
   CustomAlignedPoint,
   GeometryPointXYZ>;
 // cppcheck-suppress syntaxError - trailing comma is the only way to remove the compiler warning.
-TYPED_TEST_SUITE(PointCloudMsgWrapperTest, AllPointTypes, );
+TYPED_TEST_CASE(PointCloudMsgWrapperTest, AllPointTypes, );
```

Comment:
If you are familiar with ades, you can use our repo to set up a rolling ros2 repo in an ade. For that clone this repo and follow the instructions in the readme here: https://gitlab.apex.ai/apex.ai-public/ros2-ade

This way we could be working on the same code with the same versions and it would be easier to merge the changes.

Reply:
I don't have much left to contribute unless you want me to review specific parts of the code. I didn't change much anyway.